R/general_lineplot.R

```diff
@@ -52,7 +58,7 @@
 #' @export
 general_lineplot <- function(
   data, selected_analytes, selected_pcspec, selected_usubjids,
-  colorby_var, time_scale, yaxis_scale, cycle = NULL
+  colorby_var, time_scale, yaxis_scale, show_threshold, threshold_value, cycle = NULL
```

Comment:
suggestion: Consider adding default values for the new arguments, as they are really non-critical to the visualization.
```suggestion
  colorby_var, time_scale, yaxis_scale, show_threshold = FALSE, threshold_value = 0, cycle = NULL
```